hardware_pydantic/devices.py

```diff
@@ -56,26 +88,97 @@ def action__transfer(
             self,
             from_obj: Rack | Heater,
             to_obj: Rack | Heater,
-            transferee: Vial,
+            vial: Vial,
             to_position: str | None
     ) -> None:
         # TODO make more general
         # take it out
-        assert transferee.position_relative == from_obj.identifier
+        assert vial.position_relative == from_obj.identifier
         if isinstance(from_obj, Rack):
-            assert transferee.position in from_obj.content
-            from_obj.content[transferee.position] = None
+            assert vial.position in from_obj.content
+            from_obj.content[vial.position] = None
         elif isinstance(from_obj, Heater):
```

Comment:
Do we want this to be specific to a heater? Or is the only distinction here whether the `from_obj` is a single-position or multi-position holder?

Reply:
For the current Junior platform, the heater will heat the rack in place and no vial-moving operations are involved. I can only see its role once we have a platform with an independent heater and we will need to transfer vials to and from the heater. 

My understanding of the second question is yes. 
Please correct me if I miss understood something. Thanks. @qai222

Reply:
Let's make sure the hardware names are specific enough then -- just a `heater` implies a hotplate. Thanks for clarifying

Reply:
Thanks for the suggestion. I have added some comments for clarification.